spec/dummy/client/app/startup/ClientReduxApp.jsx

```diff
@@ -21,11 +22,20 @@ const ReduxApp = props => {
   const combinedReducer = combineReducers(reducers);
   const store = applyMiddleware(middleware)(createStore)(combinedReducer, props);
 
-  const reactComponent = (
-    <Provider store={store}>
+  let reactComponent;
+  if (Common.react013()) {
```

Comment:
I'm a bit confused that it's exposed to user land, and actually I don't understand why in 0.14 version we pass component to Redux's `Provider` instead of function, that returns component. Maybe I missed something.